### PR TITLE
Create checkbox component without semantic-ui

### PR DESCRIPTION
**Pull request by maxplt**
# Jira Issue: [IP-92](https://hedvig.atlassian.net/jira/software/projects/IP/boards/19?selectedIssue=IP-92)

## What?
Create checkbox component without semantic-ui

## Why?
For more clear using components with refs and other. And for less external libraries

## Videos

https://user-images.githubusercontent.com/89198006/135789655-9d3ddaf0-29e9-4f8b-9b66-d9555733b694.mov

https://user-images.githubusercontent.com/89198006/135789661-26b77d76-426e-4479-8b72-37f99f0949cf.mov

## Optional checklist
- [ ] Updated `src/changelog.ts`
- [ ] Codescouted
- [ ] Unit tests written
- [ ] Tested locally

### CODE REVIEW

**Comment by maxplt** [comment]
On Claims Payment I've removed strange dot in Checkbox

File: shared/@hedvig-ui/Checkbox/checkbox.tsx [hunk 1]
```diff
@@ -1,6 +1,95 @@
...
+
+export const Checkbox: React.FC<CheckboxProps> = ({
+  label,
+  checked,
+  disabled,
+  onChange,
+  className,
+  style,
+  ...props
+}) => (
```
Comment (rasmusguterstam) [inline shared/@hedvig-ui/Checkbox/checkbox.tsx:72]: I did a similar component on the `Input` component, that we might want to go for extending the native props. This way, everything will behave as expected and we'll never end up in a similar position as the weird components from semantic where refs doesn't work 🤓 

What do you think?

File: shared/@hedvig-ui/Checkbox/checkbox.tsx [hunk 2]
```diff
@@ -1,6 +1,95 @@
...
+    <CheckInput
+      className="checkbox__input"
+      tabIndex={0}
+      onKeyDown={(e) => {
+        if (e.keyCode === Keys.Enter.code) {
+          e.preventDefault()
+          // @ts-ignore
+          onChange(!e.target.checked)
+        }
+      }}
```
Comment (rasmusguterstam) [inline shared/@hedvig-ui/Checkbox/checkbox.tsx:83]: On the same note w.r.t. native props, things like these won't be an issue since the types checks out!

**Review by rasmusguterstam — APPROVED** [review]
Same here as with the `Input` component - this is amazing, and I'll approve, but check out the comment! 😄

**Pull request merged**